lib/segment/src/vector_storage/chunked_mmap_vectors.rs

```diff
@@ -178,15 +216,32 @@ impl<T: PrimitiveVectorElement + 'static> ChunkedMmapVectors<T> {
         Ok(new_id)
     }
 
-    pub fn get<TKey>(&self, key: TKey) -> &[T]
+    pub fn get<TKey>(&self, key: TKey) -> Option<&[T]>
```

Comment:
I made it optional because user may send the wrong key. Internal we just call new API for many vectors